application/modules/clients/controllers/Clients.php

```
@@ -417,16 +417,16 @@ private function validate_peppol_id(string $peppolId): void
         $this->load->model('integrations/mdl_integrations');
         $this->load->library('crypt');
 
-        $settingsService = new App\Services\Integrations\IntegrationSettingsService(
+        $settingsService = new Core\Services\Integrations\IntegrationSettingsService(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '420', column '32'). (undefined)

(MissingImport)